types/int.go

```diff
@@ -386,6 +408,11 @@ func (i Uint) IsZero() bool {
 	return i.i.Sign() == 0
 }
 
+// IsPositive returns true if Uint is positive
+func (i Uint) IsPositive() bool {
```

Comment:
When would we actually ever call this? I imagine most cases will be `if !foo.IsZero()` since it's an unsigned int.

Reply:
unnecessary removing.